Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -816,7 +816,10 @@ static RefPtr<CSSValue> consumeFontVariantAlternates(CSSParserTokenRange& range)
 
     auto parseSomethingWithoutError = [&range, &result]() {
         bool hasParsedSomething = false;
-        auto parseAndSetArgument = [&range, &hasParsedSomething] (auto& value) {
+        auto parseAndSetArgument = [&range, &hasParsedSomething] (std::optional<String>& value) {
```

Comment:
Using a `std::optional<String>` is typically rare since `String` has a null value.